code/game/objects/items/stacks/stack.dm

```diff
@@ -213,8 +213,6 @@
 		update_icon()
 		return 1
 	else
-		if(get_amount() < used)
```

Comment:
Removed these lines because they are already checked above in `can_use()` on line 205.